test/e2e/utils/utils.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Maybe name this file as launchVscodeWithMsqqlExt.ts? as it is already in utils folder.